README.md

```diff
@@ -1,66 +1,55 @@
-# Pure
+# pure [![Fish Shell Version](https://img.shields.io/badge/fish-v2.2.0-007EC7.svg?style=flat-square)](http://fishshell.com)
```

Comment:
I will go for something more like:

> # `pure` theme for `Fish` shell
[![Fish Shell Version](https://img.shields.io/badge/fish-v2.2.0-007EC7.svg?style=flat-square)](http://fishshell.com)

So it's clear it's a theme, and for fish shell.